goodale-heads-up/src/App.test.js

```diff
@@ -2,15 +2,30 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { render } from '@testing-library/react';
 import App from './App';
-import Count from './Count';
 
 describe('App', () => {
     it('renders without crashing', () => {
         const div = document.createElement('div');
         ReactDOM.render(<App/>, div);
     });
+    // it('renders Navbar Component', () => {
```

Comment:
Might be worth marking as a `TODO` so it doesn't look like you forgot to remove the commented code